grand_challenge_forge/schemas.py

```diff
@@ -83,3 +83,41 @@ def validate_pack_context(context):
         raise InvalidContextError(
             f"Invalid pack context provided:\n'{truncate_with_epsilons(context)!r}'"
         ) from e
+
+
+ALGORITHM_TEMPLATE_CONTEXT_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "algorithm": {
+            "type": "object",
+            "properties": {
+                "title": {"type": "string"},
```

Comment:
That should be slug now, right?

Reply:
No, still want a title. But I did forget to add a "slug" definition here.